admin/class-admin.php

```diff
@@ -130,24 +130,40 @@ function register_settings_page() {
 
 		$manage_options_cap = $this->get_manage_options_cap();
 
+		$notification_center     = Yoast_Notification_Center::get();
+		$notification_count      = $notification_center->get_notification_count();
+
 		// Add main page.
-		$admin_page = add_menu_page( 'Yoast SEO: ' . __( 'General Settings', 'wordpress-seo' ), __( 'SEO', 'wordpress-seo' ), $manage_options_cap, 'wpseo_dashboard', array(
+		$counter = sprintf( '<span class="update-plugins"><span class="plugin-count">%1$s</span></span>', $notification_count );
+
+		$parent_admin_page_slug = Yoast_Alerts::ADMIN_PAGE;
```

Comment:
It feels weird to have an `ADMIN_PAGE` constant in another class, I would put this constant in this class.